.fixtures.yml

```diff
@@ -1,10 +1,11 @@
 fixtures:
   repositories:
-    "puppi": "git://github.com/example42/puppi.git"
-    "monitor": "git://github.com/example42/puppet-monitor.git"
-    "firewall": "git://github.com/example42/puppet-firewall.git"
-    "iptables": "git://github.com/example42/puppet-iptables.git"
-    "concat": "git://github.com/example42/puppet-concat.git"
+    'puppi': 'git://github.com/example42/puppi.git'
```

Comment:
in metadata.json we have only stdlib as dependency.
we can get rid of puppi/monitor/firewall/iptables and maybe also concat